source/future.tex

```diff
@@ -134,6 +135,71 @@
 the namespace \tcode{std}.
 \end{example}
 
+\rSec2[ccomplex.syn]{Header \tcode{<ccomplex>} synopsis}
+\indexlibrary{\idxhdr{ccomplex}}%
+
+\begin{codeblock}
+#include <complex>
+\end{codeblock}
+
+\pnum
+The header \tcode{<ccomplex>}
+behaves as if it simply includes the header
+\tcode{<complex>}~(\ref{complex.syn}).
+
+\rSec2[cstdalign.syn]{Header \tcode{<cstdalign>} synopsis}
+
+\indexlibrary{\idxcode{__alignas_is_defined}}%
+\indexlibrary{\idxhdr{cstdalign}}%
+\begin{codeblock}
+#define @\xname{alignas_is_defined}@ 1
+\end{codeblock}
+
+\pnum
+\indexlibrary{\idxhdr{cstdalign}}%
+\indexlibrary{\idxhdr{stdalign.h}}%
+The contents of the header \tcode{<cstdalign>} are the same as the C
+standard library header \tcode{<stdalign.h>}, with the following changes:
+The header \tcode{<cstdalign>} and the header \tcode{<stdalign.h>} shall not
```

Comment:
Then we have a conundrum: both headers `<cstdbool>` and `<stdbool.h>` are deprecated, yet you want the specification to be in the main text? So the specification is not deprecated, only the use is?